<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>677</width>
         <height>532</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="imageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Info</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="imageType">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="actionclear"/>
   </widget>
   <widget class="QMenu" name="menucolor">
    <property name="title">
     <string>color</string>
    </property>
    <addaction name="actiondesaturation"/>
    <addaction name="actionnegatyw"/>
    <addaction name="actionkontrast"/>
    <addaction name="actionjasnosc"/>
    <addaction name="actionnasycenie"/>
   </widget>
   <widget class="QMenu" name="menuimage">
    <property name="title">
     <string>image</string>
    </property>
    <addaction name="actionsuma"/>
    <addaction name="actionroznica"/>
    <addaction name="actioniloczyn"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menucolor"/>
   <addaction name="menuimage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionopen"/>
   <addaction name="actionSave"/>
   <addaction name="actionclear"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionopen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/open.png</normaloff>:/res/img/open.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/save.png</normaloff>:/res/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/close.png</normaloff>:/res/img/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/img/clear.png</normaloff>:/res/img/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actiondesaturation">
   <property name="text">
    <string>desaturation</string>
   </property>
  </action>
  <action name="actionnegatyw">
   <property name="text">
    <string>negative</string>
   </property>
  </action>
  <action name="actionkontrast">
   <property name="text">
    <string>contrast</string>
   </property>
  </action>
  <action name="actionjasnosc">
   <property name="text">
    <string>brightness</string>
   </property>
  </action>
  <action name="actionnasycenie">
   <property name="text">
    <string>saturation</string>
   </property>
  </action>
  <action name="actionsuma">
   <property name="text">
    <string>sum</string>
   </property>
  </action>
  <action name="actionroznica">
   <property name="text">
    <string>difference</string>
   </property>
  </action>
  <action name="actioniloczyn">
   <property name="text">
    <string>product</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
